index.js

```diff
@@ -32,20 +32,20 @@ const slowDown = PCancelable.fn(async ({config, convert, exportOptions, inputPat
 });
 
 const slowDownService = {
-	title: 'Slow down',
+	title: 'Slow Down',
 	config: {
 		slowDownPercent: {
-			title: 'Slow down percent',
-			description: 'Percent by which to slow down the video by. For example, 0.5 means half speed, and double the duration.',
+			title: 'Slow Down Percentage',
```

Comment:
It's a bit inconsistent that this uses "percentage" while the other one uses "multiplier"